bundles/org.openhab.binding.plex/README.md

```diff
@@ -0,0 +1,172 @@
+        <Connection uri="http://192.168.1.19:32500"/>
+    </Device>
+    <Device name="SHIELD Android TV" publicAddress="XXX.XXX.XXX.XXX" product="Plex for Android (TV)" productVersion="8.26.2.29389" platform="Android" platformVersion="11" device="SHIELD Android TV" model="mdarcy" vendor="NVIDIA" provides="player,pubsub-player,controller" clientIdentifier="2c098c67afd0ca79-com-plexapp-android" version="8.26.2.29389" id="508114660" token="XXX" createdAt="1625317867" lastSeenAt="1655693424" screenResolution="1920x1080" screenDensity="320">
+        <Connection uri="http://192.168.1.6:32500"/>
+    </Device>
+    <Device name="iPad" publicAddress="XXX.XXX.XXX.XXX" product="Plex for iOS" productVersion="8.0" platform="iOS" platformVersion="14.7.1" device="iPad" model="5,3" vendor="Apple" provides="client,controller,sync-target,player,pubsub-player,provider-playback" clientIdentifier="D9629798-4B91-4375-8844-C62400573E42" version="8.0" id="617442968" token="XXX" createdAt="1646760020" lastSeenAt="1647973844" screenResolution="2048x1536" screenDensity="2">
+        <SyncList itemsCompleteCount="0" totalSize="0" version="2"/>
+        <Connection uri="http://192.168.1.220:32500"/>
+    </Device>
+    <Device name="MacBook-Pro.local" publicAddress="XXX.XXX.XXX.XXX" product="Plex for Mac" productVersion="1.41.0.2876-e960c9ca" platform="osx" platformVersion="12.2" device="" model="standalone" vendor="" provides="client,player,pubsub-player" clientIdentifier="5ehipgz2ca60ikqnv9jrgojx" version="1.41.0.2876-e960c9ca" id="507110703" token="XXX" createdAt="1625081186" lastSeenAt="1647973509" screenResolution="1680x1050,1680x1050" screenDensity=""> </Device>
+</MediaContainer>
+```
+
+Find the `Device` block of the player you want to add and fill in the `clientIdentifier` as `playerID`
+
+| Name        | Type    | Description                                                                                | Default | Required | Advanced |
+|-------------|---------|--------------------------------------------------------------------------------------------|---------|----------|---------|
+| playerID    | text    | The unique identifier of the player. `clientIdentifier` from [https://plex.tv/devices.xml] | N/A     | yes      | no      |
+
+## Channels
+
+The PLEX Player supports the following channels:
+
+| Channel              | Type   | Read/Write | Description                                                    |
+|----------------------|--------|------------|----------------------------------------------------------------|
+| currentPlayers       | String | RO         | The count of players currently configured to watch on PLEX     |
+| currentPlayersActive | String | RO         | The count of players currently being used on PLEX              |
```

Comment:
These seem rather `Number` than `String`.

Reply:
It is used in:
 
```
public ChannelUID(     @NotNull org.openhab.core.thing.ThingUID thingUID,
    String id )
```

which needs a String as argument.